lean/deps/sexpr/src/galois/sexpr.lean

```diff
@@ -117,7 +116,6 @@ def read_core {α:Type} [is_atom α] {m} [char_reader string m]
     | [] :=
       throw "unexpected close paren"
     | (top_frame::lower_frames) := do
-      consume_char,
```

Comment:
This looks wrong to me.  Try parsing `#eval (sexpr.parse "((dog) (cat)" : except string (sexpr atom))`